outputs.tf

```diff
@@ -5,7 +5,7 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # OUTPUT ALL RESOURCES AS FULL OBJECTS
 # ----------------------------------------------------------------------------------------------------------------------
-output "google_project_iam_custom_role" {
+output "google_project_iam_custom_roles" {
```

Comment:
this is a breaking change please mark it accordingly.. or revert.

Reply:
it was a typo in the readme but the plural would fit better also thinking introducing a breaking change because of this doesn't worth it, reverting

Reply:
i refactored it and made a breaking change